prody/proteins/interactions.py

```diff
@@ -182,7 +181,7 @@ def calcHydrophobicOverlapingAreas(atoms, **kwargs):
             import hpb
             imported_hpb = True
         except ImportError:
-            LOGGER.info('Please provide hpb.so file into the directory.')
+            raise ImportError('Please provide hpb.so file into the directory.')
```

Comment:
Doesn’t need to be in directory, could be compiled inside prody

Reply:
You are right. I will change that, and I will add the description in the main ProDy README.